### PR TITLE
added sample-apps

### PR DESCRIPTION
**Pull request by rominirani**

### CODE REVIEW

File: language/sample-apps/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## (Optional) Using the Custom Samples via Cloud Code for VS Code plugin
+
+If you would prefer not to use Cloud Shell and would like to utilize a Developer IDE like VS Code, we are provided support for importing and running/deploying these applications within your IDE environment. 
+
+[Cloud Code for VS Code](https://cloud.google.com/code/docs/vscode) provides IDE support for the full development cycle of Kubernetes and Cloud Run applications, from creating a cluster to running your finished application. We are providing the entire list of applications in the form of custom applications that you can import directly into VS Code in which you have configured Cloud Code. 
+
+Assuming that you have Visual Studio Code and the Cloud Code plugin setup, click on the Cloud Code link in the status bar.
+- Click on New Application
+- Select Custom Application
+- When asked for the Git Repository URL, enter the URL of this repository: https://github.com/rominirani/genai-apptemplates-googlecloud
```
Comment (polong-lin) [inline language/sample-apps/README.md:33]: Link needs updating here

File: language/sample-apps/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+| Requirement | Application Name | Technologies Used |
+|---|---|---|
+|Develop a chat application using Python Flask framework and Vertex AI PaLM API model |[chat-flask-cloudrun](chat-flask-cloudrun)|Cloud Run, Python Flask|
+|Develop a chat application using [Gradio](https://www.gradio.app/) framework and Vertex AI PaLM API model|[chat-gradio](chat-gradio)|Cloud Run, Gradio, Python|
+|Develop a chat application using [Streamlit](https://streamlit.io/) framework and Vertex AI PaLM API model|[chat-streamlit](chat-streamlit)|Cloud Run, Streamlit, Python|
+|Provide an API for the Vertex AI PaLM Code Model for your client applications.|[code-predict-cloudfunction](code-predict-cloudfunction)|Cloud Functions v2, Python|
+|Provide an API for the Vertex AI PaLM Text Model for your client applications.|[text-predict-cloudfunction](text-predict-cloudfunction)|Cloud Functions v2, Python|
+|Develop an Event-Driven application that processes uploaded files and summarizes their content|[summarization-gcs-cloudfunction](summarization-gcs-cloudfunction) |Cloud Functions v2, Cloud Storage, Python|
+
```
Comment (polong-lin) [inline language/sample-apps/README.md:23]: Could you also add an external link to the JSS solution
https://cloud.google.com/architecture/ai-ml/generative-ai-document-summarization

File: language/sample-apps/chat-flask-cloudrun/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+google-cloud-aiplatform==1.27.1
```
Comment (polong-lin) [inline language/sample-apps/chat-flask-cloudrun/requirements.txt:1]: Can you see if you can update with version >= 1.31.0?

File: language/sample-apps/chat-gradio/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+google-cloud-aiplatform==1.27.1
```
Comment (polong-lin) [inline language/sample-apps/chat-gradio/requirements.txt:1]: Could you also try to update the version here to at least 1.31.0?

File: language/sample-apps/chat-streamlit/app.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+from src.utils import *
+from src.vertex import *
+
+st.set_page_config(
+    page_title="Vertex PaLM Text Generation API",
+    page_icon=":robot:",
+    layout="centered",
+    initial_sidebar_state="expanded",
+    menu_items={
+        'Get Help': 'https://www.extremelycoolapp.com/help',
```
Comment (polong-lin) [inline language/sample-apps/chat-streamlit/app.py:12]: This URL actually exists --> could you change it to something else that definitely does not exist?

File: language/sample-apps/chat-streamlit/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,4 @@
+google-cloud-aiplatform==1.27.1
```
Comment (polong-lin) [inline language/sample-apps/chat-streamlit/requirements.txt:1]: Could you also try to update the version here to at least 1.31.0?

File: language/sample-apps/code-predict-cloudfunction/requirements.txt [hunk 7]
```diff
@@ -0,0 +1,3 @@
+functions-framework==3.*
+google-cloud-aiplatform==1.27.1
```
Comment (polong-lin) [inline language/sample-apps/code-predict-cloudfunction/requirements.txt:2]: Could you also try to update the version here to at least 1.31.0?

File: language/sample-apps/images/flaskapp-screen.png
Comment (polong-lin) [inline language/sample-apps/images/flaskapp-screen.png:1]: Should say Vertex **AI** PaLM API Chat

File: language/sample-apps/summarization-gcs-cloudfunction/requirements.txt [hunk 8]
```diff
@@ -0,0 +1,4 @@
+functions-framework==3.*
+google-cloud-storage
+google-cloud-aiplatform==1.27.1
```
Comment (polong-lin) [inline language/sample-apps/summarization-gcs-cloudfunction/requirements.txt:3]: Could you also try to update the version here to at least 1.31.0?

File: language/sample-apps/text-predict-cloudfunction/requirements.txt [hunk 9]
```diff
@@ -0,0 +1,3 @@
+functions-framework==3.*
+google-cloud-aiplatform==1.27.1
```
Comment (polong-lin) [inline language/sample-apps/text-predict-cloudfunction/requirements.txt:2]: Could you also try to update the version here to at least 1.31.0?

**Review by polong-lin — CHANGES_REQUESTED** [review]

File: language/sample-apps/README.md (hunk 2 above)
Comment (rominirani) [reply language/sample-apps/README.md:23]: Added the link to the solution.

File: language/sample-apps/chat-gradio/requirements.txt (hunk 4 above)
Comment (rominirani) [reply language/sample-apps/chat-gradio/requirements.txt]: Done

File: language/sample-apps/chat-streamlit/app.py (hunk 5 above)
Comment (rominirani) [reply language/sample-apps/chat-streamlit/app.py:12]: I have remove the Get Help and Report a bug items. The About item can remain.

File: language/sample-apps/chat-streamlit/requirements.txt (hunk 6 above)
Comment (rominirani) [reply language/sample-apps/chat-streamlit/requirements.txt]: Done

File: language/sample-apps/code-predict-cloudfunction/requirements.txt (hunk 7 above)
Comment (rominirani) [reply language/sample-apps/code-predict-cloudfunction/requirements.txt]: Done

File: language/sample-apps/summarization-gcs-cloudfunction/requirements.txt (hunk 8 above)
Comment (rominirani) [reply language/sample-apps/summarization-gcs-cloudfunction/requirements.txt]: Done

File: language/sample-apps/text-predict-cloudfunction/requirements.txt (hunk 9 above)
Comment (rominirani) [reply language/sample-apps/text-predict-cloudfunction/requirements.txt]: Done

File: language/sample-apps/chat-flask-cloudrun/requirements.txt (hunk 3 above)
Comment (rominirani) [reply language/sample-apps/chat-flask-cloudrun/requirements.txt]: Done

File: language/sample-apps/README.md (hunk 1 above)
Comment (rominirani) [reply language/sample-apps/README.md]: Done

File: language/sample-apps/images/flaskapp-screen.png
Comment (rominirani) [reply language/sample-apps/images/flaskapp-screen.png:1]: modified the index.html to have the text "Vertex AI PaLM API Chat". Updated the screen shot too.

**Review by rominirani — COMMENTED** [review]
I have made the necessary changes.

File: language/sample-apps/images/README.md
[the remark this answers is not in the document]
Comment (rominirani) [reply language/sample-apps/images/README.md:1]: Done. Renamed the folder to "assets" and updated references to it from multiple README.md files.

**Review by polong-lin — APPROVED** [review]

**Pull request merged**